js/annotations/popup.js

```diff
@@ -60,14 +60,15 @@ H.Popup.prototype = {
      * @param {HTMLDOMElement} - container where popup should be placed
      * @return {HTMLDOMElement} - return created popup's div
      */
-    init: function (parentDiv) {
+    init: function (chart) {
 
         // create popup div
         this.container = createElement(DIV, {
             className: PREFIX + 'popup'
-        }, null, parentDiv);
+        }, null, chart.container);
 
         this.lang = this.getLangpack();
+        this.iconsURL = chart.options.navigation.annotationsPopup.iconsURL;
```

Comment:
Could the `iconsURL` maybe be an option for the `Popup` constructor?
Then `Popup` would be more flexible in my opinion as it does not require a whole chart object, but just the `parentDiv` and the `iconsURL`. E.g `new Popup({ parentDiv, iconsURL })` .

Reply:
I think just config object would be nice. At this moment we have only `iconsURL` but in future, we may expect more options to pop up. Like this `new Popup({ parentDiv, { iconsURL: ..., optionA: ..., optionB: ... } })`. And user config would be: `chart.options.navigation.annotationsPopup = { iconsURL, optionA, optionB }`.

Reply:
@oysteinmoseng had a good point that keeping `parentDiv` as first parameter would make it consistent with our chart constructors. Then it would look like `new Popup(parentDiv, options)`.
It has an additional parameter, but I agree that this looks like a good solution.

What do you guys think @pawelfus and @sebastianbochan?

Reply:
Yes, of course! Just realized that I accidentally copied `new Popup({ ... , ... })`, instead of `new Popup ( ..., ... )` :)

Reply:
Great 👍 If @sebastianbochan also agrees, then we have a plan.

Reply:
Make sense for me.

Reply:
Ah, I commited changes before your discussion :) 
Will change, because config sounds really better.